networkit/_NetworKit.pyx

```diff
@@ -48,15 +48,35 @@ from .base cimport Algorithm
 from .graph cimport _Graph, Graph
 from .structures cimport _Cover, Cover, _Partition, Partition
 from .matching cimport _Matching, Matching
-from networkit.helpers cimport *
-from networkit.helpers import stdstring, pystring
 
 cdef extern from "<networkit/Globals.hpp>" namespace "NetworKit":
 
 	index _none "NetworKit::none"
 
 none = _none
 
+def stdstring(pystring):
+	""" convert a Python string to a bytes object which is automatically coerced to std::string"""
+	pybytes = pystring.encode("utf-8")
+	return pybytes
+
+def pystring(stdstring):
+	""" convert a std::string (= python byte string) to a normal Python string"""
+	return stdstring.decode("utf-8")
+
+cdef extern from "cython_helper.h":
+	void throw_runtime_error(string message)
+
+cdef extern from "<algorithm>" namespace "std":
+	void swap[T](T &a,  T &b)
+	_Graph move( _Graph t ) nogil # specialized declaration as general declaration disables template argument deduction and doesn't work
+	_Partition move( _Partition t) nogil
+	pair[_Graph, vector[node]] move(pair[_Graph, vector[node]]) nogil
+	vector[pair[pair[node, node], double]] move(vector[pair[pair[node, node], double]]) nogil
+	vector[double] move(vector[double])
+	vector[bool_t] move(vector[bool_t])
+	vector[pair[node, node]] move(vector[pair[node, node]]) nogil
+
```

Comment:
Why is this re-introduced into `_NetworKit.pyx`?

Reply:
This was a mistake I made initially - I thought I could import these functions from outside of `_NetworKit.pyx` but I could not. In commit db6d351 the changes are reverted.